src/Component/Metadata/Index.php

```diff
@@ -36,6 +36,9 @@ public function __construct(
         ?bool $write = null,
         ?string $formType = null,
         ?array $formOptions = null,
+        ?string $stateMachineComponent = null,
+        ?string $stateMachineTransition = null,
+        ?string $stateMachineGraph = null,
```

Comment:
I'm quite sure, we don't want to support state machine transition on Index operation